pom.xml

```diff
@@ -141,9 +145,11 @@
         <module>extensions-core/protobuf-extensions</module>
         <module>extensions-core/lookups-cached-global</module>
         <module>extensions-core/lookups-cached-single</module>
+        <module>extensions-core/ec2-extensions</module>
```

Comment:
This is definitely a nice change. The EC2 autoscaling stuff is better suited to an extension than to core. Hopefully it opens the door to removing Druid's dependency on `aws-common`.